backup/re_initialise.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# (Re-)initialise a system from a backup that has been dumped into the root tree.
+
+chown mastodon:mastodon /opt/mastodon/mastodon.env
+
+source /opt/mastodon/versions.env
+chmod 0444 /opt/backup/mastodon_db_backup.sql
+
+echo "Importing mastodon database"
+# We run this in the background instead of using --detach so the log output remains visible
+/usr/bin/docker run --rm  \
+					  --shm-size=256mb \
+					  --health-cmd="pg_isready -U postgres" \
+					  --env POSTGRES_HOST_AUTH_METHOD=trust \
+					  --volume=/opt/backup/mastodon_db_backup.sql:/docker-entrypoint-initdb.d/mastodon_db.sql \
+					  --volume=/opt/mastodon/postgres:/var/lib/postgresql/data \
+					  --name=mastodon-db-import \
+					  "${POSTGRES_IMAGE}" &
```

Comment:
That's a hell of indentation. :)

Reply:
AAAAAaah! Someone did not set "tabs are 4 spaces, and tabs are spaces" when writing this... Good catch!